src/patternfly/components/Dropdown/dropdown-multi-select.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="pf-c-dropdown__menu" role="dialog" aria-labelledby="{{id}}" {{#unless pf-c-dropdown--IsExpanded}}hidden="hidden"{{/unless}}>
+    <form class="pf-c-form">
+    <fieldset class="pf-c-form__fieldset">
```

Comment:
if we are going to include a fieldset here should we add the legend as well? It may require a `pf-u-sr-only`